cost-analyzer/values.yaml

```diff
@@ -62,6 +62,9 @@ saml: # enterprise key required to use
         assertionvalues:
           - "readonly"
 
+imagePullSecrets:
```

Comment:
This should be commented out by default. Can you also test to make sure nothing regresses when this is completely missing?